rero_ils/modules/items/serializers/response.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Serialization response factories.
+
+Responsible for creating a HTTP response given the output of a serializer.
+"""
+
+from __future__ import absolute_import, print_function
+
+from datetime import datetime
+from flask import current_app
+
+from invenio_records_rest.serializers.response import add_link_header
+
+
+def search_responsify_csv(serializer, mimetype):
+    """Create a Records-REST search result response serializer.
+
+    :param serializer: Serializer instance.
+    :param mimetype: MIME type of response.
+    :returns: Function that generates a record HTTP response.
+    """
+
+    def view(pid_fetcher, search_result, code=200, headers=None, links=None,
+             item_links_factory=None):
+        response = current_app.response_class(
+            serializer.serialize_search(pid_fetcher, search_result,
+                                        links=links,
+                                        item_links_factory=item_links_factory),
+            mimetype=mimetype)
+        response.status_code = code
+        if headers is not None:
+            response.headers.extend(headers)
+        file_name = '{date}-inventory.csv'.format(
+            date=datetime.now().strftime('%Y%m%d')
```

Comment:
In RERO ILS I think we talk about using aware date with timezone (in case we work in a different timezone).

Do you think it's possible to use the timezone from the library in which we do the inventory?

For now we use this method for the timezone date: https://github.com/rero/rero-ils/blob/dev/rero_ils/modules/libraries/api.py#L317

Info: the timezone can change the date. For example if the server is in Switzerland, and you make an inventory in New York at 6'30PM, the date wouldn't be the same.